Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -84,6 +90,20 @@ static inline D2D_RECT_F __CGRectToD2D_F(CGRect rect) {
 
     // Alpha Blending
     CGFloat alpha = 1.0f;
+
+    inline void ComputeStrokeStyle(ID2D1Factory* factory) {
```

Comment:
>ComputeStrokeStyle [](start = 16, length = 18)

Premature optimization: It might make sense to take the ID2D1DeviceContext here and query the factory only if you already dont have the stroke style.  Will simplify calling code, and save some cycles when you dont need to re-compute.